src/ArrayFromString.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @since 0.5.17
+ */
+import * as t from 'io-ts'
+import { pipe } from 'fp-ts/lib/pipeable'
+import { map, Traversable } from 'fp-ts/lib/Array'
+import { chain, Applicative } from 'fp-ts/lib/Either'
+
+/**
+ * @since 0.5.17
+ */
+export interface ArrayFromStringC<C extends t.Mixed> extends t.Type<Array<t.TypeOf<C>>, string, unknown> {}
+
+/**
+ * @example
+ * import { ArrayFromString } from 'io-ts-types/lib/ArrayFromString'
+ * import { NumberFromString } from 'io-ts-types/lib/NumberFromString'
+ * import { right } from 'fp-ts/lib/Either'
+ *
+ * assert.deepStrictEqual(ArrayFromString(NumberFromString, ",").decode("1,2,3"), right([1,2,3]))
+ * assert.deepStrictEqual(ArrayFromString(NumberFromString, /[\s,]+/).decode("1, 2, 3"), right([1,2,3]))
+ * @since 0.5.17
+ */
+export function ArrayFromString<C extends t.Mixed>(
+  codec: C,
```

Comment:
This will not work for all codecs you need to gurantee that the output of the used `Type` is `string`.

Something like `t.Type<A, string>` should be used instead.